docs/source/configuration_file.rst

```diff
@@ -143,8 +143,8 @@ Loader Parameters
     {
         "$schema": "http://json-schema.org/draft-04/schema#",
         "title": "path_data",
-        "description": "Path of the BIDS folder.",
-        "type": "string"
+        "description": "Path(s) of the BIDS folder(s).",
+        "type": "list"
```

Comment:
could also be str (based on meeting discussion)

Reply:
Fixed here: https://github.com/ivadomed/ivadomed/pull/588/commits/b50aaeb46abfec8595237388c1c6eb4e84d9979a